cmd/bbolt/README.md

```diff
@@ -0,0 +1,325 @@
+# Introduction to bbolt command line
+
+Bbolt is a tool for inspecting and examining bbolt databases by accessing the low level details of the bbolt database. <br>
+`bbolt` is a command line client for Bbolt databases. To install bbolt command-line please refer [here](https://github.com/etcd-io/bbolt#installing). 
+
+**Note**: [Etcd](https://github.com/etcd-io/etcd) uses bbolt database as its backend storage engine. In this document, we will be 
+analysing etcd's bbolt database to inspect data in etcd cluster, if you don't want to use it then you can use [this](https://github.com/etcd-io/bbolt#opening-a-database) to create/open a bbolt database.
+
+1. Start a single member etcd cluster with this command:
+    ```
+    $etcd --data-dir default.etcd
+    ```
+
+    It will create a `data-dir` with name `default.etcd` and directory structure will look like this:
+    ```
+    ~/tree default.etcd
```

Comment:
Suggest consistency with trailing `$` on bash commands.  Either use it for all, or none.

```suggestion
    tree default.etcd
```